.yarn/patches/@react-native-menu-menu-npm-1.2.1-99faaa4b8d.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/android/src/main/java/com/reactnativemenu/MenuView.kt b/android/src/main/java/com/reactnativemenu/MenuView.kt
+index 7841d0c7f0a7ba8576013ea632ed042ee3b056e2..17ed7c6491b60e87fe829fb3f8841424617b4cfb 100644
+--- a/android/src/main/java/com/reactnativemenu/MenuView.kt
++++ b/android/src/main/java/com/reactnativemenu/MenuView.kt
+@@ -53,7 +53,7 @@ class MenuView(private val mContext: ReactContext) : ReactViewGroup(mContext) {
+     updateTouchDelegate()
+   }
+
+-  override fun onInterceptTouchEvent(ev: MotionEvent?): Boolean {
```

Comment:
This patch is necessary to compile on Android. I've opened a PR in the react-native-menu repo: https://github.com/react-native-menu/menu/pull/1022
Once it gets merged we can remove this patch.

Reply:
@compojoom looks like it was merged.